internal/tui/components/inputbox/inputbox.go

```diff
@@ -74,6 +120,15 @@ func (m Model) View() string {
 		)
 }
 
+func (m Model) ViewWithAutocomplete() string {
+	baseView := m.View()
+	autocompleteView := ""
+	if m.autocomplete != nil {
+		autocompleteView = m.autocomplete.View()
+	}
+	return baseView + "\n" + autocompleteView
```

Comment:
Let's also show the autocomplete keys in this component's help menu alongside Ctrl+D etc.